.github/workflows/atd_moped_build_stage.yml

```diff
@@ -15,16 +15,17 @@ jobs:
     name: Build Editor
     runs-on: ubuntu-latest
     steps:
+      # Get the code first
+      - name: "Checkout"
+        uses: actions/checkout@v2
```

Comment:
the python step (below) does not need this step, but i learned the [hard way](https://stackoverflow.com/questions/64405836/github-actions-no-such-file-or-directory-on-any-run-step) that you need to `checkout` before you can access files in your repo 🤦